.github/workflows/package-release.yml

```diff
@@ -0,0 +1,113 @@
+name: "Package Release"
+run-name: "Package Release ${{ github.ref_name }}"
+
+on:
+  push:
+    tags:
+      - "*"
+
+jobs:
+  build-pecl:
+    name: "Create PECL package"
+    runs-on: "ubuntu-latest"
+
+    steps:
+      - name: "Checkout"
+        uses: "actions/checkout@v4"
+        with:
+          # Manually specify a ref. When actions/checkout is run for a tag without a ref, it looks up the underlying
+          # commit and specifically fetches this to the refs/tags/<tag> ref, which denies us access to the tag message
+          ref: ${{ github.ref }}
+          fetch-depth: 2
+          submodules: true
+
+      - name: "Build Driver"
+        uses: ./.github/workflows/linux/build
+        with:
+          version: "8.3"
+
+      - name: "Write changelog file for packaging"
+        run: git tag -l ${{ github.ref_name }} --format='%(contents)' > changelog
+
+      - name: "Build package.xml"
+        run: "make package.xml"
+
+      - name: "Build release archive"
+        run: "make package"
+
+      # PECL always uses the version for the package name.
+      # Read it from the version file and store in env to use when uploading artifacts
+      - name: "Read current package version"
+        run: echo "PACKAGE_VERSION=$(./bin/update-release-version.php version)" >> "$GITHUB_ENV"
+
+      # Currently fails with: Cannot install, php_dir for channel "pecl.php.net" is not writeable by the current user
+      # - name: "Install release archive to verify correctness"
+      #  run: pecl install mongodb-${{ env.PACKAGE_VERSION }}.tgz
```

Comment:
Using `sudo` fixes this. I also changed the extensions we install when setting up PHP to avoid `pecl install` failing because the extension is already loaded.